test/codeu/chat/TestRunner.java

```diff
@@ -17,22 +17,89 @@
 import org.junit.runner.JUnitCore;
 import org.junit.runner.Result;
 import org.junit.runner.notification.Failure;
+import java.io.*;
 
 public final class TestRunner {
-  public static void main(String[] args) {
-     final Result result =
-         JUnitCore.runClasses(
-             codeu.chat.common.SecretTest.class,
-             codeu.chat.common.UuidTest.class,
-             codeu.chat.common.UuidsTest.class,
-             codeu.chat.relay.ServerTest.class,
-             codeu.chat.server.BasicControllerTest.class,
-             codeu.chat.server.RawControllerTest.class,
-             codeu.chat.util.store.StoreTest.class
-         );
-      for (final Failure failure : result.getFailures()) {
-         System.out.println(failure.toString());
+
+  private static boolean loading = true;
+  private static int FRAME_SPEED = 60;
+
+  /**
+   * updated to a different thread as not to interfere
+   * with the loading message animation
+   *
+   * @param msg: the message to be displayed in the loading
+   */
+  public static synchronized void main(String[] msg) {
+    Thread th = new Thread() {
```

Comment:
`Thread thread = new Thread() {` please for better readability through this lengy-ish method.